medusa/databases/failed_db.py

```diff
@@ -24,8 +24,8 @@ def execute(self):
         queries = [
             ('CREATE TABLE failed (release TEXT, size NUMERIC, provider TEXT);',),
             ('CREATE TABLE history (date NUMERIC, size NUMERIC, release TEXT, provider TEXT,'
-             ' old_status NUMERIC DEFAULT 0, showid NUMERIC DEFAULT -1, season NUMERIC DEFAULT -1,'
-             ' episode NUMERIC DEFAULT -1);',),
+             ' status NUMERIC DEFAULT 0, quality NUMERIC DEFAULT 0, showid NUMERIC DEFAULT -1',
```

Comment:
Should `status` default to `-1` instead since that's the default status? https://github.com/pymedusa/Medusa/blob/feature/add-db-status-quality-fields/medusa/common.py#L103

Reply:
Yes. You're right.